<?xml version="1.0" encoding="UTF-8"?><testsuite failures="1" name="cucumber.runtime.formatter.JUnitFormatter" skipped="0" tests="1" time="109.368238">
<testcase classname="Application Login Feature" name="Application Login Test Scenario" time="109.368238">
<failure message="java.lang.IndexOutOfBoundsException: Index: 1, Size: 1&#13;&#10;&#9;at java.util.ArrayList.rangeCheck(Unknown Source)&#13;&#10;&#9;at java.util.ArrayList.get(Unknown Source)&#13;&#10;&#9;at java.util.Collections$UnmodifiableList.get(Unknown Source)&#13;&#10;&#9;at stepDefinitions.LoginStepDefinitionDataTable.user_Enter_Username_and_Password(LoginStepDefinitionDataTable.java:50)&#13;&#10;&#9;at ✽.Then User Enter username and password(C:/Users/hiteshg/git/CucumberPractise/CucumberPractiseProject/src/main/java/Features/LoginDataTable.feature:6)&#13;&#10;"><![CDATA[Given User is already on Login Page.........................................passed
When Title of Login page is Pyd.............................................passed
Then User Enter username and password.......................................failed
Then User clicks on Login button............................................skipped
And User is on Home Page....................................................skipped
Then close the browser......................................................skipped

StackTrace:
java.lang.IndexOutOfBoundsException: Index: 1, Size: 1
	at java.util.ArrayList.rangeCheck(Unknown Source)
	at java.util.ArrayList.get(Unknown Source)
	at java.util.Collections$UnmodifiableList.get(Unknown Source)
	at stepDefinitions.LoginStepDefinitionDataTable.user_Enter_Username_and_Password(LoginStepDefinitionDataTable.java:50)
	at ✽.Then User Enter username and password(C:/Users/hiteshg/git/CucumberPractise/CucumberPractiseProject/src/main/java/Features/LoginDataTable.feature:6)
]]></failure>
</testcase>
</testsuite>
